custom_components/better_thermostat/climate.py

```diff
@@ -596,11 +606,11 @@ def hvac_action(self):
 	def target_temperature(self):
 		"""Return the temperature we try to reach."""
 		# if target temp is below minimum, return minimum
-		if self._target_temp < self._min_temp:
-			return self._min_temp
+		if self._target_temp < self._TRV_min_temp:
```

Comment:
No, we should check against our own boundaries, not against the boundaries of the TRV for our target temperature.